src/app/components/book-details/book-details.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { Book } from 'src/app/books';
+
+@Component({
+  selector: 'app-book-details',
+  templateUrl: './book-details.component.html',
+  styleUrls: ['./book-details.component.css']
+})
+export class BookDetailsComponent implements OnInit {
+  
+  @Input() book?: Book;
+    constructor() { }
+  
+
+  ngOnInit(): void {
```

Comment:
if empty, can be removed

Reply:
This is component for another part implementation and can be filled after in another PR(for book details)